### PR TITLE
chore: enable gzip compression in nginx

### PR DESCRIPTION
**Pull request by bararchy**
One of the issues we saw in the wild was a problem with gzip responses. It would be nice to have Broken Crystals using compression.

### CODE REVIEW

File: nginx-config/default.conf [hunk 1]
```diff
@@ -7,7 +7,10 @@ server {
     ssl_certificate_key /etc/letsencrypt/live/brokencrystals.com/privkey.pem;
 
     root         /var/www/html;
-
+    # Add Gzip support
+    gzip on;
+    gzip_types      text/plain application/xml application/json;
```
Comment (derevnjuk) [inline nginx-config/default.conf:12]: Enable the compression for the static content as well

```suggestion
    gzip_types
        application/xml
        application/json
        font/eot
        font/otf
        font/ttf
        image/svg+xml
        text/css
        text/javascript
        text/plain
        text/xml;
```

File: nginx-config/default.conf [hunk 2]
```diff
@@ -7,7 +7,10 @@ server {
     ssl_certificate_key /etc/letsencrypt/live/brokencrystals.com/privkey.pem;
 
     root         /var/www/html;
-
+    # Add Gzip support
+    gzip on;
```
Comment (derevnjuk) [inline nginx-config/default.conf:11]: Disable the compression for small files 

```suggestion
    gzip on;
    gzip_min_length 256;
```

**Review by derevnjuk — CHANGES_REQUESTED** [review]
Please apply the same changes to two configuration files: `./public/nginx-config/default.conf` and [nginx-proxy-config.yaml](https://github.com/NeuraLegion/cluster-vulnerable-apps/blob/main/vulnerable-apps/helm/charts/brokencrystals/templates/nginx-proxy-config.yaml) in the helm. These changes should be identical in both files.

File: nginx-config/default.conf (hunk 1 above)
Comment (bararchy) [reply nginx-config/default.conf:12]: seems for static files we need something like: 
```
location / {
    gzip_static on;
}

```

Comment (derevnjuk) [reply nginx-config/default.conf:13]: If I'm not mistaken `gzip_static` is for precompressed files

**Pull request closed without merging**